api/tests/acceptance/application/organizations-administration/organization-administration-get_test.js

```diff
@@ -75,6 +75,8 @@ describe('Acceptance | Routes | organization-administration-controller', functio
               type: organization.type,
               'logo-url': organization.logoUrl,
               'external-id': organization.externalId,
+              'parent-organization-id': organization.parentOrganizationId,
+              'parent-organization-name': null,
```

Comment:
Peut être adapter le test pour refléter la situation.

```suggestion
              'parent-organization-name': 'Nom Orga',
```

Reply:
Le parentOrganizationId est null aussi - je n'ai pas modifié cela parce que cette orga est archivée donc dans cette situation le lien avec l'orga mère a été supprimé. Et comme l'id du parent est null, j'ai ajouté un parent-organization-name null aussi...

Reply:
Ok